vigetlabs/storyboard
# [#107] Allow users to create custom themes for their stories

## Addresses #107 

- Allows users to create custom themes to be used in their stories.

## To Test

- Log into Storyboard
- Click on `My Themes`
- Click on `Create a Custom Theme`
- Fill out form with title and any other fields you want.
- Save theme.
- Click on `My Stories`
- Click on `Create a Story`
- Switch theme to `Custom`
- Select your custom theme
- Fill out the rest of the form
- Save the story
- Navigate to the story's show page
- Confirm it has your custom theme.

### Screenshots
![Screen Shot 2022-07-08 at 2 17 29 PM](https://user-images.githubusercontent.com/25255820/178049506-fec52f5d-9375-4904-b4b4-045692d327ec.png)
![Screen Shot 2022-07-08 at 2 17 42 PM](https://user-images.githubusercontent.com/25255820/178049507-33c5b24e-80db-4187-93dc-d191b08fe5e0.png)
![Screen Shot 2022-07-08 at 2 17 58 PM](https://user-images.githubusercontent.com/25255820/178049509-e0ba8165-8e8d-46f6-a11d-4c976b86e240.png)
![Screen Shot 2022-07-08 at 2 18 12 PM](https://user-images.githubusercontent.com/25255820/178049512-73a9dab8-4ca3-4a29-b2d1-2dcab585f6bf.png)
![Screen Shot 2022-07-08 at 2 18 39 PM](https://user-images.githubusercontent.com/25255820/178049514-20913af2-add6-4aa0-a771-0dad24754f1c.png)
![Screen Shot 2022-07-08 at 2 18 49 PM](https://user-images.githubusercontent.com/25255820/178049589-94a4574c-26ef-49a9-bea2-18fd08bb3252.png)
![Screen Shot 2022-07-08 at 2 19 12 PM](https://user-images.githubusercontent.com/25255820/178049593-30b4b2e3-85fa-465f-984d-cbdeb77e6712.png)
